src/resources/lang/en/seat.php

```diff
@@ -107,15 +107,25 @@
     'notification_enable'             => 'Notify to Slack',
     'integration_channel'             => 'Integration Channel',
 
-    'slack_integration' => 'Slack Integration',
-    'enabled'           => 'Enabled',
-    'default_channel'   => 'Default Channel',
-    'webhook'           => 'Webhook',
-    'emoji_full'        => 'Full Emoji',
-    'emoji_half'        => 'Half Emoji',
-    'emoji_empty'       => 'Empty Emoji',
-    'help_emoji'        => 'Setup which emoji to use to display the "importance" of an operation when relaying to Slack.',
-    'save'              => 'Save',
+    'notifications_to_send' => 'Notifications to send',
+
+    'help_notify_operation_interval' => 'Decide how many pings to send before each operation. Each value is the number of minutes prior to the operation to send the ping. Separate numbers with commas. Default value of "15,30,60" will send 3 notifications: 15 minutes, 30 minutes, and 60 minutes prior to the operation start time.',

```

Comment:
well done, maybe just add the value in a `code` tag so it's be more visible and put it as a translation parameter inside the view.
- https://laravel.com/docs/8.x/localization#replacing-parameters-in-translation-strings

Reply:
<img width="405" alt="Screen Shot 2022-01-01 at 12 26 50 PM" src="https://user-images.githubusercontent.com/70491080/147856382-2a0601d2-9735-41a7-a6e0-d44839a38da0.png">